spec/requests/api/v1/articles_spec.rb

```diff
@@ -0,0 +1,144 @@
+require 'rails_helper'
+
+RSpec.describe 'Articles', type: :request do # rubocop:disable Metrics/BlockLength
+  headers = { 'Authorization': "Token token=#{ENV['HOWITZER_TOKEN']}" }
+
+  describe 'GET #index' do
```

Comment:
Question about naming. Looks like it is got from testing controllers. I would not recommend that approaches. The question is what kind of test do we use (unit or integration).
Examples:

```ruby
# unit testing
RSpec.describe XxxController do
  describe '#index'
  describe '#show'
end
```

```ruby
# integration/acceptance testing

RSpec.describe "API V1 Bathrooms", type: 'request' do
  describe "POST /api/v1/bathrooms" do
    context "with valid parameters" do
      it "creates a new bathroom" do
      end
    end
end
```

I recommend to implement acceptance tests instead of units where we test API as users it does

Reply:
done